### ISSUE TITLE
What Anti Block are for ?

### ISSUE THREAD

**Issue by ExDomino** [TheCricket/Chisel-2#468]
What Anti Block are for? I thought they was supposed to be a light source, but they emit only 1 of light. What are their purpose? Thank you in advance.

**Comment by Drullkus** [comment]
They're themed after a game called [Antichamber](https://www.youtube.com/watch?v=aGsnm2nOnso).

**Comment by ExDomino** [comment]
They make no light?

**Comment by CptSpaceToaster** [comment]
Correct, they make no light, but are rendered without the light-map being applied to them either.  They do not emit 1 block of light. They simply aren't effected by shadows or normal light sources so the texture is very bright!

**Comment by Drullkus** [comment]
I haven't seen you in forever, @CptSpaceToaster :)

And yes, exactly what he said.
